lib/EnsEMBL/REST/Model/Lookup.pm

```diff
@@ -258,6 +259,19 @@ sub find_object_by_stable_id {
   return $self->find_object($id, $species, $object_type, $db_type);
 }
 
+sub fetch_remarks_by_transcript {
+  my ($self, $object, $code) = @_;
+  my $object_id;
+  if (defined($object)) {
+    $object_id = $object->dbID();
+  }
+  my $c = $self->context();
+  my $db_type = $c->request->param('db_type') || 'core';
```

Comment:
Logically speaking we should use // and not || here, shouldn't we? I mean, it isn't particularly like that we would like to call our database type "0" but in theory we _could_ do it.